AlfrescoApp/Utils/Categories/AlfrescoNode+Sync.h

```diff
@@ -16,7 +16,8 @@
  *  limitations under the License.
  ******************************************************************************/
 
-#import "AlfrescoNode.h"
+#import <AlfrescoSDK-iOS/AlfrescoSDK.h>
```

Comment:
I'm surprised this is needed. The `AlfrescoSDK.h` file should be included as part of the precompiled header.